hole/play.go

```diff
@@ -161,14 +161,12 @@ func Play(ctx context.Context, holeID, langID, code string) (score Scorecard) {
 	// Args
 	switch langID {
 	// FIXME brainfuck and fish should be consistent.
-	case "brainfuck":
+	case "brainfuck", "fish", "hexagony":
```

Comment:
I think this will break hexagony, atm we're passing the src over stdin and the args over argv and the interpreter is doing the args to stdin mangling, I think.

Reply:
That's correct. The interpreter will need an update. So will hexagony.net's "Arguments" input mode but I could handle that once it's changed here.

Reply:
@GrayJoKing I updated the Hexagony interpreter if you want to switch the version to 4b30f58 in your commit.

https://github.com/SirBogman/Hexagony/commit/4b30f58d3fe2ab0c602cb5d4dfd2ba07022209cf

Reply:
One more thing, you will have to update the code on hole-ng.js that generates a link to https://hexagony.net.